src/main/java/org/opensearch/security/ssl/transport/SecuritySSLRequestHandler.java

```diff
@@ -108,21 +104,7 @@ public final void messageReceived(T request, TransportChannel channel, Task task
         }
 
         try {
-
-            Netty4TcpChannel nettyChannel = null;
-
```

Comment:
curious to understand why these changes were made?

Reply:
To get rid of type casting